MC/config/QC/json/trd-tracking-task.json

```diff
@@ -0,0 +1,48 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
```

Comment:
all these `not_applicable` entries can be removed

Reply:
I'm keeping them because all other JSON files have the same structure, and I'm unsure if they might be needed later

Reply:
But we should try to keep the json files as compact as possible. It makes maintenance easier. So please remove them. If there are other TRD QC json files where you find this you could even clean them up there as well :)

Reply:
done